ansible/roles/prosody/files/util.internal.lib.lua

```diff
@@ -213,6 +213,18 @@ function Util.has_prefix(str, prefix)
     return str:sub(1, #prefix) == prefix
 end
 
+function Util.starts_with_one_of(str, prefixes)
```

Comment:
Can we drop this and use the one from jitsi-meet?
https://github.com/jitsi/jitsi-meet/blob/371b9a3e95477ee8bfe94420c0ff8b14159b9625/resources/prosody-plugins/util.lib.lua#L284